pallets/stellar-relay/src/tests.rs

```diff
@@ -105,7 +105,7 @@ fn create_valid_dummy_scp_envelopes(
 	txes.push(transaction_envelope.clone()).unwrap();
 	let transaction_set = TransactionSet { previous_ledger_hash: Hash::default(), txes };
 
-	let tx_set_hash = crate::compute_non_generic_tx_set_content_hash(&transaction_set);
```

Comment:
Same here, I think we need to use a `match` or something

Reply:
I added `unwrap` method but now i replace it with 
`.expect("Should compute non generic tx set content hash");`